editoast/src/views/v2/timetable/stdcm.rs

```diff
@@ -217,13 +219,45 @@ async fn stdcm(
         time_gap_after: data.time_gap_after,
         margin: data.margin,
         time_step: Some(2000),
+        work_schedules: build_work_schedules(conn, data.start_time).await?,
     }
     .fetch(core_client.as_ref())
     .await?;
 
     Ok(Json(stdcm_response))
 }
 
+async fn build_work_schedules(
+    conn: &mut DbConnection,
+    time: DateTime<Utc>,
+) -> Result<Vec<SimpleWorkSchedule>> {
+    let res = Ok(WorkSchedule::list(conn, Default::default())
+        .await?
+        .iter()
+        .map(|ws| {
+            let schedule = SimpleWorkSchedule {
+                start_time: delta_as_ms(&ws.start_date_time, &time),
+                end_time: delta_as_ms(&ws.end_date_time, &time),
```

Comment:
Why do we need to send work_schedules with `end_date_time < time`?

Reply:
We don't need to, I've added a filter. In fact we can also filter out the ones that start too late